README.md

```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.org/abought/demo-ember-osf.svg?branch=develop)](https://travis-ci.org/abought/demo-ember-osf)
 [![Coverage Status](https://coveralls.io/repos/github/abought/demo-ember-osf/badge.svg?branch=develop)](https://coveralls.io/github/abought/demo-ember-osf?branch=develop)
+[![Known Vulnerabilities](https://snyk.io/test/github/abought/demo-ember-osf/badge.svg)](https://snyk.io/test/github/abought/demo-ember-osf)
```

Comment:
I'm willing to give this a shot- looks interesting.

(In conjunction with this, I'll need to get around to finishing the rollout of greenkeeper as well, and making sure it respects pinning to an ember LTS. The real test will be how well the two auto-upgrade services play together.)

....Though at the rate we're going, we're going to need to turn this into a yeoman generator just to make sure all the badge URLs get updated...!!)